tests/test.py

```diff
@@ -71,19 +71,19 @@ def test_session(capability):
 
 if __name__ == "__main__":
    chrome_on_windows_11 = {
+    "browserName" : "Chrome",
+    "browserVersion" : "latest",
     'bstack:options' : {
       "os" : "Windows",
       "osVersion" : "11",
-      "browserVersion" : "latest",
       "projectName" : "My Project",
       "buildName" : "test percy_screenshot",
       "sessionName" : "BStack first_test",
       "local" : "false",
-      "seleniumVersion" : "3.14.0",
+      "seleniumVersion" : "4.0.0",
       "userName": USER_NAME,
-      "accessKey": ACCESS_KEY,
-    },
-    "browserName" : "Chrome",
+      "accessKey": ACCESS_KEY
```

Comment:
```suggestion
      "accessKey": ACCESS_KEY
```
- Access key should be inside ?

Reply:
You can validate it from here: https://www.browserstack.com/automate/capabilities